docs/rules/no-useless-spread.md

```diff
@@ -12,6 +12,7 @@
   - Spread an array literal as elements of an array literal
   - Spread an array literal as arguments of a call or a `new` call
   - Spread an object literal as properties of an object literal
+  - Use spread syntax to clone an immediate array
```

Comment:
How should we call this? It's a newly created array, can't be used anywhere else.